ci/test_integration.sh

```diff
@@ -157,6 +157,18 @@ done
 rm -rf exp dump data
 cd "${cwd}" || exit 1
 
+# [ESPnet2] test enh recipe
```

Comment:
Thank you! Could you also add validation part for the config files?

https://github.com/espnet/espnet/blob/933c94870a2ed6b1748fb34dbd13406b61f65a75/ci/test_integration.sh#L164-L177

Reply:
Done.